### ISSUE TITLE
skipRendering true throws exception

### ISSUE THREAD

**Issue by ayZagen** [tscanlin/tocbot#309]
If `skipRendering` is true tocbot throws following error:
```txt
build-html.js:199 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'querySelector')
    at Object.updateToc (build-html.js:199:41)
    at Object.<anonymous> (index.js:197:17)
    at Object._scrollListener (index.js:79:12)
    at tocbot.init (index.js:207:10)
    at AsyncFunction.OnThisPage_component_useVisibleTask_JUUUE8xYwDo [as resolved] (on-this-page.tsx:106:12)
    at AsyncFunction.invoke (use-core.js:59:26)
    at qrl-class.js:70:35
    at then (promises.js:20:56)
    at qrl-class.js:56:20
    at use-task.js:439:27
```

It throws this error in this line 
```js
var oldActiveTocLink = tocElement.querySelector('.' + options.activeLinkClass)
```

It seems `tocElement` is undefined, however I am sure it is defined and I can see it successfully resolved in the `tocbot.init` function.

Do I miss something? Thanks.

**Comment by tscanlin** [comment]
Hey, thanks for opening this issue! And apologies for my delay in getting back to you.
I have a PR [here](https://github.com/tscanlin/tocbot/pull/310) that should address this. Let me know if that fix doesn't work or if you run into anything else. Thanks!

**Comment by tscanlin** [comment]
This should be fixed in [4.21.2](https://github.com/tscanlin/tocbot/commit/e03244c57f8c9719cf86babaf73a27c5c98b650d)
